swiftsimio/__init__.py

```diff
@@ -31,8 +47,22 @@ def mask(filename, spatial_only=True) -> SWIFTMask:
     Sets up a masking object for you to use with the correct units and
     metadata available.
 
+    Parameters
+    ----------
+    filename : str
+        SWIFT data file to read from
+    spatial_only : bool, optional
+        flag for specifing spatial masking, default to True
```

Comment:
Flag for only spatial masking, this is much faster but will not allow you to use masking on other variables (e.g. density). Defaults to ``True``.